src/content/en/2022/cdn.md

```diff
@@ -0,0 +1,376 @@
+
+CDNs can provide better performance for delivering non-static content as well as they often optimize the routes and use most efficient transport mechanisms, but we see the usage of serving the HTML still lags considerably behind the other two types.
+
+{{ figure_markup(
+  image="cdn-usage-hosted-comparison.png",
+  caption="Trends for content served from CDN for mobile",
+  description="This chart shows the trends for content served from CDN for last few years. The general trend is that the CDN usage is increasing. For the contents served from subdomains we see bigger increase.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vRZQqyyKiQWgICD_a0WaEAmfZyFN_Zi3wVuQnZxsXqwZ_1JQg2x7GpRw4CTkX4gKvurzoCQ6YokkdpM/pubchart?oid=2141710369&format=interactive",
+  sheets_gid="2085205637",
+  sql_file="top_cdns.sql"
+  )
+}}
+
+Compared to the [year 2021](../2021/cdn) we found that the usage of CDN has been steadily increasing. There was a large bump in CDN usage for the content served from subdomains, and a smaller one for HTML while third-party CDN usage remained relatively static.
+
+These are some of the potential reasons that can be attributed to this rise:
+
+* Post pandemic, many businesses took a large portion of their physical business online. This put a lot of strain on their servers and found that it was much more efficient to server the static content through CDNs for offloading through caching and faster delivery.
+* This increase was not seen in 2021 as many businesses were still trying to figure out the optimal solution for their problem. In fact we saw a dip in CDN usage for the subdomain and third-party type.
+* Sites relied on serving third-party content through third-party domains instead of their own domains. The fact that the amount of content served from third-party domains increased by 3% during this period supports this assumption.
+
+Regarding the base HTML page, the traditional pattern has been to serve the base HTML from the origin and this pattern has continued as majority of base pages continue to be served from the origin. However, there has been a 4% increase in the base pages being served from CDNs. The trend of base HTML pages being served from the CDN is clearly on the rise.
+
+These are some of the likely reasons behind the rise:
+* CDNs can improve load time of the base HTML page that can be of high importance to improve customer experience and keep users engaged.
+* It is simpler and faster to load the base page from CDNs using the DNS provided by CDN providers. {# TODO - is it? Why? #}
```

Comment:
@harendra @joeviggiano I've a query on this statement.

Reply:
@tunetheweb Thanks for reviewing this. Regarding your query, let me put the reason, Distributed nature of CDNs make recursive query response faster.
(https://www.cloudflare.com/learning/dns/what-is-1.1.1.1/).  For eg. Amazon R53 does not require you to CNAME to CloudFront reducing extra DNS hops, this also adds to the performance. If you want to serve the site via apex domain you generally need to host it on DNS provided by CDN, for the users it's simpler to just host their domains in CDN providers.

Reply:
@tunetheweb Thanks, yea that works.